lib/ramble/ramble/util/logger.py

```diff
@@ -114,6 +117,14 @@ def _stream_kwargs(self, default_kwargs={}, index=None):
 
         return kwargs
 
+    @contextmanager
+    def configure_colors(self, **kwargs):
+        old_value = llnl.util.tty.color.get_color_when()
+        if 'stream' in kwargs:
+            llnl.util.tty.color.set_color_when('never')
+        yield
+        llnl.util.tty.color.set_color_when(old_value)
```

Comment:
Since the 'set_color_when(old_value)' is not nested in an 'else' and is not indented under 'yield', would this method always overwrite the 'if' and set it back to 'old_value' instead of 'never'?

Reply:
Yes. So, if we print without an active stream the logic here looks like:

```
get old value
(print stuff)
set value to old value
```

but the logic when a stream is active changes it to:
```
get old value
set value to never
(print stuff)
set value to old value
```